### ISSUE TITLE
Running Angular Universal --> Memory Heap Allocation

### ISSUE THREAD

**Issue by CharlieGreenman** [nrwl/nx#14054]
UPDATE: LOOK AT END AT HOW WAS ABLE TO SOLVE (MIGHT HELP)

### Current Behavior

https://stackoverflow.com/questions/74951123/angular-universal-node-javascript-memory-heap-still-failing-after-upping-max-ol

### Expected Behavior

That command should run as expected

### Steps to Reproduce

1. large repo
2. Add commands 

### Nx Report

```shell
NA
```

### Failure Logs

```shell
NA
```

### Additional Information

NA

**Comment by CharlieGreenman** [comment]
reference: https://github.com/nrwl/nx/issues/1110

**Comment by CharlieGreenman** [comment]
UPDATE: 

Was able to solve via the following: 

```
"ng": "node --max-old-space-size=8192 ./node_modules/.bin/ng",
"nx": "node --max-old-space-size=8192 ./node_modules/.bin/nx",
"start:serve": "npm run nx -- run zeta-frontend:serve",
```

That finally propagated it as expected.

**Comment by CharlieGreenman** [comment]
Scratch that, it solved but then went back to not solving.

**Comment by CharlieGreenman** [comment]
I found the issue. Being that angular universal makes a large initial bundle on serve, it would cause the development bundle to fail silently unless budget was updated. By doing the following: 

```
"budgets": [
                {
                  "type": "initial",
                  "maximumWarning": "20mb",
                  "maximumError": "40mb"
                },
```

even on my development build, that solved the issue. This should really make it's way over to the angular repo as better error reporting for angular universal.
